reflections/2020-05-22.md

```diff
@@ -0,0 +1,29 @@
+# Reflections
+
+## Participation
+
+- 3 groups 6-8 people in size.
+- Each group had a brainhack committee member and a python/programming enthusiast/expert
+
+## What Happened
+
+- some groups were quiet from the onset and didn't have questions to open up discussion
+- one group had the brainhack member share their screen and run through the lesson
+- another group had questions about what editor they should use (spyder/jupyter notebooks)
+- some participants did not know they could edit their google colab notebooks
+- some participants are working ahead, and some are sticking to the schedule
+
+## Take-Away(s)
+
+- offer more guidance on how participants can interact/use their google colab notebooks
+- during discussion, try having one of the group leaders or participants share their screen
+  to manipulate the notebook used in the episode to encourage question generation.
+- have brainhack members generate some questions as they work through that week's material
+  - they can pose those questions to their groups for that week to help start discussion.
```

Comment:
My thoughts here are that it might be fine to just openly isolate an aspect of the lesson and call on people (after asking for volunteers) to explain parts of code.... maybe make up something new with their guidance to see if their understanding of the code replicates? If it doesn't problem-solve with the group to figure out what the error was, and why it happened?